src/Pages/ChannelRequests.razor

```diff
@@ -147,9 +147,14 @@
                             </Feedback>
                         </NumericPicker>
                         <FieldHelp>
-
-                            @($"Amount in Satoshis. Minimum {_minimumChannelCapacity:f8}. Current amount: {Math.Round(PriceConversionService.SatToUsdConversion(new Money(_amount, MoneyUnit.BTC).Satoshi, _btcPrice), 2)} USD")
-
+                             @if (Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT < _amount)
```

Comment:
Here should be something like max channel size. But this is being validated by the validator in line 143.

Reply:
We will have a crash if I will remove this implementation ,"Exception":"System.ObjectDisposedException: Cannot access a disposed object.\n   at Microsoft.AspNetCore.Components.RenderTree.ArrayBuilder`1.GrowBuffer(Int32 desiredCapacity)\n

Reply:
Change the variable for the max channel size, please. Also I think this should be checked only if the network is regtest.

Reply:
Also, errors should not be on field help, they should be added as errortext in the validator helper. Please fix this.

BTW I've never liked the idea of having a helper for validations unless widely used.